### DISCUSSION TITLE
Problème de dimensionnement d'un modal avec composite

### DISCUSSION THREAD

**Discussion by Mathis554** [witsa/synapps#746] [Q&A] [answered]
Bonjour,
J'affiche un modal qui me permet d'affiche un composite planning.
![image](https://github.com/witsa/synapps/assets/174346872/3dca4216-f864-491f-9c2f-8b5d0384d308)
Quand je lance mon synapps dans un grand format, je vois entièrement mon ensemble modal et composite, mais quand je lance mon synapps sur un écran plus petit, j'ai mon ensemble qui ne s'affiche pas entièrement malgré le fait que j'ai mis l'option Dépassement Horizontal & vertical en automatique
![image](https://github.com/witsa/synapps/assets/174346872/88d62c3d-afd8-49c6-b256-7addeb3dff8f)

**Comment by JeremieDENAEGEL** [comment, marked as answer]
Bonjour,

C'est probablement due aux tailles entrées dans tes acteurs.
Essaies de passer sur des tailles vmin pour tes acteurs ou alors tu pourrais mettre tes acteurs en enfant d'un acteur "boite à vue" qui aurait une taille en vmin ou vh/vw

**Reply by Mathis554** [reply]
Merci, je vais essayer sa ce matin.

**Reply by Mathis554** [reply]
Merci beaucoup, j'ai pu réussir. 
![image](https://github.com/user-attachments/assets/b98d4703-e6f1-4a3d-b632-1f41f5259fb4)
![image](https://github.com/user-attachments/assets/abcf7e16-f0aa-461f-a05e-7de537fe464b)
En plus, maintenant, j'ai les barres pour pouvoir naviguer.
